react/ProductDescription.tsx

```diff
@@ -55,14 +66,47 @@ function ProductDescription(props: PropsWithChildren<Props>) {
   const { handles } = useCssHandles(CSS_HANDLES, { classes: props.classes })
   const intl = useIntl()
   const { product } = useProduct()
+  const { isMobile } = useDevice()
 
+  const { collapseContent = true, title, customCollapse = false } = props
   const description = props.description ?? product?.description
+  const [isCustomCollapsed, setCustomCollapsed] = useState(customCollapse)
+
+  const {
+    desktopLimitCharacters = defaultDesktopCharacters,
+    mobileLimitCharacters = defaultMobileCharacters,
+  }: any = props
```

Comment:
Any particular reason you typed it as `any`? This should be avoided

Reply:
I'll remove that `any` . 
Please reply to the other message to know if I also remove the useRef and commit after.